snapshot/bundlestore/groupcache.go

```diff
@@ -73,8 +73,8 @@ func toPeers(nodes []cluster.Node, stat stats.StatsReceiver) []string {
 		peers = append(peers, "http://"+string(node.Id()))
 	}
 	log.Info("New groupcacheStore peers: ", peers)
-	stat.Counter("peerDiscoveryCounter").Inc(1)
-	stat.Gauge("peerCountGauge").Update(int64(len(peers)))
+	stat.Counter("peerDiscoveryCounter").Inc(1)		// TODO errata metric - remove if unused
+	stat.Gauge("peerCountGauge").Update(int64(len(peers)))	// TODO errata metric - remove if unused
```

Comment:
We're casting too wide a net for errata if we're not making super important stats like this first-class. I was thinking that errata would be the somewhat redundant or trivial stuff and that we'd have a separate section for standard metrics that were useful but weren't absolutely required as key indicators.